docs/reference/logging/splunk.md

```diff
@@ -32,16 +32,27 @@ You can set the logging driver for a specific container by using the
 You can use the `--log-opt NAME=VALUE` flag to specify these additional Splunk
 logging driver options:
 
-  - `splunk-token` required, Splunk HTTP Event Collector token
+  - `splunk-token` required, Splunk HTTP Event Collector token.
```

Comment:
@vbatts  This really belongs in a table. Hard to read in this format. I recommend:

http://www.tablesgenerator.com/markdown_tables